vmm/Cargo.toml

```diff
@@ -39,6 +39,7 @@ hex = { version = "0.4.3", optional = true }
 hypervisor = { path = "../hypervisor" }
 igvm = { version = "0.3.1", optional = true }
 igvm_defs = { version = "0.3.1", optional = true }
+landlock = "0.4"
```

Comment:
This should be a precise 3 digit version - dependabot bumps versions for us.

Reply:
fixed